projects/systemml/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://github.com/apache/systemml"
+primary_contact: "janardhan@apache.org"
+auto_ccs:
+  - "janardhan@apache.org"
```

Comment:
No need to repeat same email in auto-cc. Add the mailing list as you cited in comment.